tests/protocols/test_websocket.py

```diff
@@ -744,13 +746,11 @@ async def websocket_session(uri):
         port=unused_tcp_port,
     )
     async with run_server(config):
-        task = asyncio.create_task(
-            websocket_session(f"ws://127.0.0.1:{unused_tcp_port}")
-        )
+        asyncio.create_task(websocket_session(f"ws://127.0.0.1:{unused_tcp_port}"))
```

Comment:
the `task =` is important please keep it

Reply:
It's important when you don't want the GC to clean it, since it's a weak ref, but in this case since all the logic happens in the same function scope, I don't think I need it?

Reply:
it's been a while so I dont remember exactly but when I reworked the whole test suite and added strict settings to pytest this helped a lot.